mmv1/third_party/terraform/provider/provider.go.erb

```diff
@@ -255,7 +255,8 @@ func DatasourceMapWithErrors() (map[string]*schema.Resource, error) {
 		"google_compute_backend_service":                   compute.DataSourceGoogleComputeBackendService(),
 		"google_compute_backend_bucket":                    compute.DataSourceGoogleComputeBackendBucket(),
 		"google_compute_default_service_account":           compute.DataSourceGoogleComputeDefaultServiceAccount(),
-		"google_compute_disk":        					    compute.DataSourceGoogleComputeDisk(),
+		"google_compute_disk":                              compute.DataSourceGoogleComputeDisk(),
+		"google_compute_region_disk":                       compute.DataSourceGoogleComputeRegionDisk(),
```

Comment:
nit: this list is in alphabetical order. Would you mind moving `google_compute_region_disk` after `google_compute_regions`? Thanks!